etest_test/qemu_test.py

```diff
@@ -0,0 +1,55 @@
+"""Tests for etest.qemu."""
+
+import logging
+
+import docker
+import pytest
+
+import etest.qemu as sut
+
+logger = logging.getLogger(__name__)
+
+
+def test_init() -> None:
+    """Ensure QEMU only opens on non-amd64 architectures."""
+    assert sut.qemu(logger, "amd64").get_enabled() is False
```

Comment:
This is why I suggest _enabled instead of __enabled.  Tests can peek inside of implementation details to verify what they need to.  Does this test things that are true by definition though?  If so, then no need for this test.

Reply:
What do you mean by "true by definition"?

Reply:
True by definition means that the source code under test defines the value that you're testing it defines.  Rather than being calculated in some strange fashion.  Assigning a format string would be true by definition, using standard python operators to compose something would be true by definition (someone else is proving they have that behaviour).  Let me know if that's clear or if I should expand more.